### DISCUSSION TITLE
Transform: Organize fields should allow adding arbitrary values and keep old ones

### DISCUSSION THREAD

**Discussion by roidelapluie** [grafana/grafana#39221] [Dashboards, Panels & Visualizations]
![org](https://user-images.githubusercontent.com/291750/133408219-d6783861-b4db-4195-bd1a-5dc6f4b92b11.png)

I am using an outer join followed by "Organize fields"

I use organized fields to rename the HTTP status codes to proper names. Depending on the time or the variables, not all the status codes are available all the time.

When we edit the panel, we lose the fields that are not present at the time of editing. This is annoying. In the same way, we can predict more values, but are not able to "pre-set" them in advance.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
